frontend/src/pages/Player/Toolbar/NewCommentForm/CommentTextBody/mentions.module.css

```diff
@@ -73,14 +73,4 @@
 
 .mentions__suggestions__item--focused {
 	background: var(--color-gray-300);
-}
-
-.mentions__mention {
```

Comment:
Was not being used cause called incorrectly [here](https://github.com/highlight/highlight/pull/8307/files#diff-38b7695022c228eabe463c67d45bea7f706cbee8d5f0d8a8b65861a77fed9d8eL151). When applied it caused incorrect spacing in the text area.